ga4gh/exceptions.py

```diff
@@ -595,3 +552,25 @@ class RepoManagerException(Exception):
     """
     Signals something went wrong inside the repo manager
     """
+
+
+class DuplicateNameException(RepoManagerException):
+    """
+    The user has tried to insert an object with the same name as
+    an existing object.
+    """
+    def __init__(self, objectName, containerName=None):
+        msg = "Name '{}' already exists".format(objectName)
+        if containerName is not None:
+            msg += " in '{}'".format(containerName)
+        super(DuplicateNameException, self).__init__(msg)
```

Comment:
Well, names are still required to be unique _within_ a container, but they don't have to be unique between containers. This is a generic exception that we use to report when someone tries to insert a (say) dataset with the same name as an existing dataset.  I've added a comment to clarify this point.